### ISSUE TITLE
Changelog

### ISSUE THREAD

**Issue by dideler** [jonasschmidt/ex_json_schema#53]
Is there already a way to see what's changed between versions? I couldn't find a changelog or release notes on the GitHub Releases page.

We've recently encountered some breaking changes in production when upgrading.

**Comment by jonasschmidt** [comment]
Yes, good point. There were some breaking changes from `0.5.x` to `0.6.0`, mostly resulting from functions that were intended for internal use being used as public API because there was no officially documented way of validating fragments/sub-schemas.

The email validation was broken by a regression and should have been fixed in `0.7.1` with this commit: https://github.com/jonasschmidt/ex_json_schema/commit/43e36b0e7a8630d35d29f85a272fefc80fe0d8fd. Is it still failing for you with the current version?

I will add a changelog and document at least the breaking changes between versions.

**Comment by dideler** [comment]
Thanks. I forgot to point out that we initially upgraded from 0.5.7 to 0.6.2 when we had the issues.  
We're now upgrading to version 0.7.3 and will let you know if we encounter any issues.
